src/back/services/discord.ts

```diff
@@ -218,7 +218,7 @@ export class DiscordService {
     if (DISCORD_SERVICE_ENABLED) {
       inBackground(async () => {
         try {
-          const publicUpdates = getPublicUpdates(await UpdateModel.find<UpdateAttributes>({ proposal_id: proposalId }))
+          const publicUpdates = getPublicUpdates(await UpdateService.getAllByProposalId(proposalId))
```

Comment:
shouldn't we change this so newUpdate messages also uses projects instead of proposals?

Reply:
yes! but I would do it in another pr :)